src/App.tsx

```diff
@@ -1,11 +1,50 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  FormEvent,
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { addTodos, deleteTodo, getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/Page/ErrorNotification';
+import { Footer } from './components/Page/Footer';
+import { TodoList } from './components/Page/TodoList';
+import { TodoContext } from './components/TodoContext';
+import { TodosError } from './types/ErrorEnum';
+import { FilterType } from './types/FilterTypeEnum';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useContext(TodoContext);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.All);
+  const [title, setTitle] = useState<string>('');
+  const [todosError, setTodosError] = useState<TodosError>(TodosError.None);
+  const [isAdding, setIsAdding] = useState<boolean>(false);
+
+  if (todosError.length > 0) {
+    setTimeout(() => {
+      setTodosError(TodosError.None);
+    }, 3000);
+  }
```

Comment:
You need to move this logic to `ErrorNotifications` component.
And create setTimeout on the first render and clear timeout on unmount